config/application.rb

```diff
@@ -24,5 +24,6 @@ class Application < Rails::Application
     config.generators.system_tests = nil
     config.time_zone = 'Australia/Melbourne'
     config.active_record.legacy_connection_handling = false
+    if !ENV['ALLOWED_HOSTS'].empty? then config.hosts << ENV['ALLOWED_HOSTS'] end
```

Comment:
Rails has [middleware to prevent DNS rebinding](https://www.bigbinary.com/blog/rails-6-adds-guard-against-dns-rebinding-attacks). This code chunk allows adding `ALLOWED_HOSTS` via env files. If the var is not set, then default settings for localhost are used.